src/app/app.component.ts

```diff
@@ -12,30 +15,18 @@ import { AdaptService } from "./adapt.service";
 })
 
 export class AppComponent {
-    @ViewChild(DxResponsiveBoxComponent) box: DxResponsiveBoxComponent;
     adaptOptions: any;
-    constructor(private adapt: AdaptService, router: Router) {
+    constructor(private adapt: AdaptService,
+        @Inject(DOCUMENT) _document: any,
+        private transferState: TransferState,
+        @Inject(PLATFORM_ID) private platformId: any) {
         this.adapt.adapt$.subscribe(item => {
             this.adaptOptions = this.adapt.getOptionsForAdaptation(item);
         });
-        router.events.subscribe((val) => {
-            if(val instanceof NavigationEnd){
-                this.box.instance.repaint();
-            }
-        });
-    }
-    adaptation() {
-        this.adapt.setAdaptValue();
-    }
-    getScreen() {
-        let width = window.innerWidth;
+        this.adapt.setAdaptValue(_document.documentElement.clientWidth);
```

Comment:
I suppose if we use `documentElement.clientWidth` here, we should use it instead of the `window.innerWidth` in resize handler too.